code/modules/overmap/exoplanets/_exoplanet.dm

```diff
@@ -55,7 +56,6 @@
 	var/list/possible_themes = list(
 		/datum/exoplanet_theme = 30,
 		/datum/exoplanet_theme/mountains = 100,
-		/datum/exoplanet_theme/radiation_bombing = 10, 
```

Comment:
???

Reply:
Missed the removal part of a whitespace change. Will fix.